pino/pino.d.ts

```diff
@@ -0,0 +1,62 @@
+// Type definitions for pino v1.0.1
+// Project: https://github.com/mcollina/pino.git
+// Definitions by: Peter Snider <https://github.com/psnider>
+// Definitions: https://github.com/borisyankov/DefinitelyTyped
+
+/// <reference path="../node/node.d.ts"/>   // only for require()
+
+declare module 'pino' {
+
+    import stream = require('stream')
+
+    interface Serializers {
+        req(req: Object): Object
+        res(res: Object): Object
+    }
+
+
+    interface LoggerOptions {
+        name?: string
+        enabled?: boolean
+        safe?: boolean
+        serializers?: Serializers
+    }
+
+
+    interface Pino {
+        (): Logger
+        (options: LoggerOptions): Logger
+        (stream: stream.Writable): Logger
+        (options: LoggerOptions, stream: stream.Writable): Logger
+    }
+
+
+
+    interface Logger {
+        child(bindings: Object): this
```

Comment:
Please do not use `Object`. either `{}` or `any` depending on the context. here `{}` would be the correct fit.